liquibase-core/src/main/java/liquibase/util/StringUtils.java

```diff
@@ -88,12 +88,80 @@ public static String[] processMultiLineSQL(String multiLineSQL, boolean stripCom
 
         String trimmedString = StringUtils.trimToNull(currentString.toString());
         if (trimmedString != null) {
-            returnArray.add(trimmedString);
+            if (!returnArray.isEmpty() && isLastStatementComment(parsed, trimmedString)) {
```

Comment:
There is the `stripComments` argument passed to this function, but this ends up stripping off the comment even if stripComments=false, right? We should make sure this overall function is respecting that flag passed along.

Reply:
Yes, that's correct if comment is stripped by SqlParser then this logic will just NOT find the comment and it will work correctly, but it will take time to go through this logic, so I will add additional condition to this if statement, thanks!